dojo/tools/contrast/parser.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, filename, test):
             impact = "N/A"
             references = "N/A"
 
-            dupe_key = hashlib.md5(category + '|' + str(cwe) + '|' + title + '|').hexdigest()
+            dupe_key = hashlib.md5(category.encode('utf-8') + str(cwe).encode('utf-8') + title.encode('utf-8')).hexdigest()
```

Comment:
Quite sure this won't be needed in python 3 anymore ?
@Maffooch can you confirm ?
If so , I think we can try to sneak this into the python 2 branch ?

Reply:
Yes, we are still pushing patches to 2 as well.

Reply:
@adracea These changes are very much needed in python3 branch but not in python2 branch. There was some sneaky unicode changes in python3.